lib/mix/tasks/ecto.tenant.migrate.ex

```diff
@@ -153,6 +153,8 @@ defmodule Mix.Tasks.Ecto.Tenant.Migrate do
       Task.async_stream(tenants, fn tenant ->
         create_tenant(tenant)
 
+        repo.set_tenant(tenant.name)
```

Comment:
What does setting the tenant help here?

Reply:
This is the controversial part. In order to simplify the code, we broke how `put/get_dynamic_repo` works by default, which some of these Ecto tasks rely on.